server/api-util/lineItems.js

```diff
@@ -182,7 +182,7 @@ exports.transactionLineItems = (listing, orderData, providerCommission) => {
   };
 
   // Note: extraLineItems for product selling (aka shipping fee)
-  //       is not included to commission calculation.
+  //       is not included in either customer or provider commission calculation.
```

Comment:
Perhaps there should be a bit longer description of what provider vs customer commissions are - and how they affect the payin and payout.